backend/game/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.urls import path
+from django.contrib import admin
+from django.urls import path, include
+from django.conf.urls import url, include
+from django.contrib.auth.models import User
 
 from . import views
 
 urlpatterns = [
-    path('dice/', views.views_rest.roll_dice),
-]
+    url(r'^',views.FrontendAppView.as_view()),
```

Comment:
I won't know for certain until this is all merged and done with, but, with those this routes, it looks like it will be easy to add Channels to this.